lib/go-tc/deliveryservices.go

```diff
@@ -241,15 +241,20 @@ func (ds *DeliveryServiceNullableV12) Sanitize() {
 	}
 }
 
-func getTypeName(tx *sql.Tx, id int) (string, bool, error) {
+func getTypeData(tx *sql.Tx, id int) (string, string, bool, error) {
```

Comment:
Mind adding a Godoc function comment here, explaining the return params? That way someone doesn't have to read the function to figure out what each string being returned is.

Reply:
done